project/BuildSettings.scala

```diff
@@ -31,6 +32,18 @@ object BuildSettings {
       "-Ywarn-unused", // necessary for Scalafix RemoveUnused rule (not present in sbt-tpolecat for 2.13)
       "-P:silencer:checkUnused"
     ),
+    missinglinkExcludedDependencies ++= List(
```

Comment:
Why are all these dependencies excluded? Isn't the point of all of this to check the compatibility with everything?

Reply:
you can see the detailed logs here
https://gist.github.com/sideeffffect/4c82347ae93053acac6999b6b055a473

they seem to me as essentially false positives -- I guess it's not rare in the Java world to somewhere in your code call to a class/method that might not always be available, typically for logging libraries

Reply:
Alright.

What is the strategy on how to find out whether the report is false positive or not? I am afraid that this might be a problem and lots of false positives would make it too hard to release this project and I don't want that.

Reply:
I can't say I'm an expert, I guess we will have to learn as we go.
But from my experience, missinglink has found for me on binary incompatibility that in Circe, which after I suppressed the PFs from Java logging libraries, has stuck out quite obviously.
Probably because it's a Scala thing, which I know has recently released breaking change versoin.

A bad sign would be if we had to start suppressing Scala (let alone TypeLevel) libs. But so far it's only low level Java things and there have never been problems with binary incompatibility with them -- it's always Scala things, mostly TypeLevel jars.

Reply:
The fact that there are considerably many FPs is just because sst aggregates sooo many libs. Even a big project (like the one I work on internally) has just a few.